flent/runners.py

```diff
@@ -680,8 +680,17 @@ def find_binary(self, **args):
             args['socket_timeout'] = ""
 
         if args['test'] in ("TCP_STREAM", "TCP_MAERTS", "omni"):
+            self.ctrl_port = args['netperf_control_port']
+            ss_args = {'host': self.remote_host or 'localhost',
+                       'interval': args['interval'],
+                       'length': args['length'],
+                       'target': args['host'],
+                       'ip_version': args['ip_version']}
+
+            self.add_child(SsRunner, self.delay, None, ss_args)
```

Comment:
No, ss_args is the 'command' of the processrunner. That is overloaded, so that a dict passed as that parameter will be expanded into kwargs for the find_binary function.